### DISCUSSION TITLE
[vcpkg_script] Customize port configure variables (CMake) from triplet.

### DISCUSSION THREAD

**Discussion by Neumann-A** [microsoft/vcpkg#19248] [Ideas]
Nice feature to have to change the configure options via the triplet file instead of touching the portfile. Currently only additional compiler flags can be set if I read the scripts correctly.

**Comment by cbezault** [comment]
@Neumann-A it's not exactly clear to me what you're trying to describe here.
What variables are you trying to customize from the triplet?

**Comment by Neumann-A** [comment]
something like VCPKG_CMAKE_ADDITIONAL_OPTIONS which passes additional options to cmake based ports in the configure step. (Just like VCPKG_CXX_FLAGS allows passing extra CXX_FLAGS)

**Comment by cbezault** [comment]
So effectively cmake defaults?
